### ISSUE TITLE
Error "This store is not configured to use this ledger"

### ISSUE THREAD

**Issue by pif2000** [btcpayserver/btcpayserver#1134]
Hello

Im trying to make a transaction sending bitcoins from btcserver wallet to ledger nano S and im getting this error message "This store is not configured to use this ledger."  I have also tried transffer small amount (<9$ 12$ then full amount 970) none of them worked. 

BTCPayServer v1.0.3.127
Ledger Live 1.15.0

Can anyone help me with this problem please?

**Comment by pavlenex** [comment]
Well we have to narrow it down a little bit:
- Is this the first time you're sending funds from your btcpay with Ledger?
- Are you sure that you're using a proper store? Do you have more than one store in your BTCPay?
- Are you sure that the addresses displayed on your BTCPay on your Ledger matched when you initially configured?

**Comment by pif2000** [comment]
Thanks for answering Pavlenex. Is it indeed my first time when transfering funds from btcpay with Ledger and im a noob to bitcoin world. Initially after i setup the server i have blockchain wallet associated with my btcpay server. Because of the gap limit my funds where NOT transffer in blockchain wallet and they remains in btcpay server wallet. This is why i have purchased ledger nano s so i can transffer funds there. I have follow this tutorial https://www.youtube.com/watch?v=1Sj5mP4TkFI and i have managed to changed blockchain wallet to ledger nano s wallet.
I have only one store and its setup properly, its working well.
Not sure i understand your last question, can you please explain more ?

**Comment by pavlenex** [comment]
Just to understand better, are you trying to spend the funds that you received to your Blockchain info wallet?

**Comment by pif2000** [comment]
Sorry for my poor explanation. The funds from btcpay store transactions are 0.10380165 btc and for some reason they were not automatically transfered into my blockchain wallet as they were in past. I simple forgot to check bitcoins transactions for couple of months. When i have checked my blockchain wallet i noticed amount similar with 0.2 $ or something like this. Basically in my btcpay server is shows up a different funds than blockchain wallet shows up. 
I have tried to use setting Manage wallet: Send -> to my blockchain account and is NOT working. Thats why i have purchased ledger nano s.

**Comment by pavlenex** [comment]
Okay, so if I understood correctly the error you're getting then makes sense.
What you're trying to do is sign a transaction with a **wrong wallet.**
In order to spend funds received to your blockchain.info store, you cannot use Ledger S.
What you are basically trying to do is tell Ledger, hey confirm that you own this private key from blockchain.info, since that's not true, you need to sign with a key that belongs to your blockchain.info.

What you should do is:

## Configure a brand new store

1. Create a new store in your BTCPay, call it `Ledger Store` for example.
2. [Set up your Ledger wallet](https://www.ledger.com/start/) if you already have not.
3. [Follow this video from start ](https://www.youtube.com/watch?v=1Sj5mP4TkFI)on how to configure your Ledger wallet in BTCPay.

Now you should have a new store that you'll use with your Ledger Nano S. This store will have zero balance in your wallet.

## Now, we need to transfer funds from your old store and blockchain.info wallet.
Basically we will extract the keys from your blockchain info and sign transaction with a private key, we will send funds to our newly created Ledger Nano S Store.

1. Go to your blockchain.info and get a **private key**. I am not sure how you can do this. You need to do a research.
2. In your BTCPay, go to Invoices > Create a new invoice. Select **Ledger Store**.
3. Open your BlockChain Info Store > Wallets > Send. 
4. Copy the amount and address 
5. Click `Sign with mnemonic or private key`
![2019-11-07_23-18-11](https://user-images.githubusercontent.com/36959754/68432583-0272ae80-01b5-11ea-8aeb-4d0f511c1449.png)
6. Once you sign a transaction it should all be good.

**Comment by pavlenex** [comment]
I just realized blockchain.info is probably not using BIP39.

In this case, disregard the above steps, download Electrum wallet and try to r[ecover your blockchain.info wallet in Electrum](https://bitcointalk.org/index.php?topic=2360742.0), by selecting `Legacy` wallet. Once you do that, you need to [customize the gap limit.](https://github.com/btcpayserver/btcpayserver-doc/blob/master/ElectrumWallet.md#configuring-the-gap-limit-in-electrum).

When you recover the wallet and customize the gap limit, you should see your funds. Then you can send them to your Ledger Nano S store that you will configure as described in the steps above.

Moral of the story **DO NOT USE BLOCKCHAIN.INFO WALLET.**

**Comment by pif2000** [comment]
Pavlenex thanks so much for information i appreciate them. Yes you are right about not using blockchain wallet, this wallet is good at the begining when i use to have three transactions in a month, once the store grows up then the 20 gap limit is insuficient and thats when problem starts.

**Comment by pavlenex** [comment]
One thing you should not forget is that if you want to use your new Ledger Nano S store, you need to re-pair it with whatever e-commerce software plugin you're using.

If I misunderstood your issue, feel free to come to our chat.btcpayserver.org and ping me @pavlenex so we can quicker figure out what's the issue.

**Comment by pavlenex** [comment]
Ok, I used blockchain.info for the first time in my life to be sure how it works.

I assume you backed that wallet up, if you have not you can do that in **Security Center**.

Then you import that private key that will look like this `tree other window force time name verify dilemma better proof truck author` as a legacy wallet into Electrum and configure gap limit.

In screenshoots it would look like this
![2019-11-07_23-59-12](https://user-images.githubusercontent.com/36959754/68435031-b3c81300-01ba-11ea-80a4-c59ad2c8d49e.png)
![Screenshot 2019-11-07 at 23 55 34](https://user-images.githubusercontent.com/36959754/68435046-c2162f00-01ba-11ea-9478-48d2f00d12cc.png)
![Screenshot 2019-11-07 at 23 55 41](https://user-images.githubusercontent.com/36959754/68435050-c2162f00-01ba-11ea-95c2-d2e5e55cfff6.png)
![Screenshot 2019-11-07 at 23 56 27](https://user-images.githubusercontent.com/36959754/68435052-c2162f00-01ba-11ea-9af9-03cc4c14e177.png)
![Screenshot 2019-11-07 at 23 56 34](https://user-images.githubusercontent.com/36959754/68435053-c2aec580-01ba-11ea-8817-5847e00b9bca.png)
![Screenshot 2019-11-07 at 23 56 41](https://user-images.githubusercontent.com/36959754/68435054-c2aec580-01ba-11ea-9536-8cf848739660.png)

After the wallet is recovered, [configure the gap limit as explained in Electrum docs.](https://github.com/btcpayserver/btcpayserver-doc/blob/master/ElectrumWallet.md#configuring-the-gap-limit-in-electrum)

**Comment by pif2000** [comment]
I managed to perform the second option you mentioned and changed the actual wallet with electrum wallet. I also managed to changed the gap limit from 20 to 100. Im able to see my funds now! 
Since i bought the Ledger S to sove this problem, and it was not neccesarly, i will try in the morning to connect it to my store for better protection. @pavlenex thanks so much for all your explanations, tutorials and for your help!

**Comment by pavlenex** [comment]
Great!

I'll go ahead and close this issue since we solved it.
Your next step would be creating a brand new store with Ledger, or replacing the derivation scheme in your old store with Ledgers. That's the easier part since it's all well-documented and even has a step by step video.
